src/main/java/org/fogbowcloud/sebal/SEBALHelper.java

```diff
@@ -60,83 +45,78 @@
 import org.opengis.referencing.operation.TransformException;
 
 public class SEBALHelper {
-	
+
 	private static Map<Integer, Integer> zoneToCentralMeridian = new HashMap<Integer, Integer>();
-	
+
 	private static final Logger LOGGER = Logger.getLogger(SEBALHelper.class);
-	
-    public static Product readProduct(String mtlFileName,
-            List<BoundingBoxVertice> boundingBoxVertices) throws Exception {
-        File mtlFile = new File(mtlFileName);
-        LandsatGeotiffReaderPlugin readerPlugin = new LandsatGeotiffReaderPlugin();
-        LandsatGeotiffReader reader = new LandsatGeotiffReader(readerPlugin);
-        return reader.readProductNodes(mtlFile, null);
-    }
-
-    public static BoundingBox calculateBoundingBox(List<BoundingBoxVertice> boudingVertices,
-            Product product) throws Exception {
-        List<UTMCoordinate> utmCoordinates = new ArrayList<UTMCoordinate>();
-    	
-        MetadataElement metadataRoot = product.getMetadataRoot();
-        
+
```

Comment:
What is this exception. When is it thrown?

Reply:
This is thrown by the method _readProductNodes_.